plugins/CoreAdminHome/Commands/InvalidateReportData.php

```diff
@@ -223,4 +239,40 @@ private function getSegmentsToInvalidateFor(InputInterface $input, $idSites)
         }
         return $result;
     }
+
+    private function getSegmentsIdToInvalidateFor(InputInterface $input, $idSites)
+    {
+        $idSegments = $input->getOption('idsegments');
+        $idSegments = trim($idSegments);
+        $idSegments = preg_replace('/\s/', '', $idSegments);
+        $idSegments = explode(",", $idSegments);
+        $idSegments = array_map('trim', $idSegments);
+        $idSegments = array_unique($idSegments);
+
+        if (empty($idSegments)) {
+            return array(null);
+        }
```

Comment:
What's the reason for returning `[null]`? Should it instead return an empty array so nothing is invalidated? Given it's unexpected we could also output a warning here.